src/main/java/io/gravitee/el/spel/TemplateParserContext.java

```diff
@@ -25,7 +25,7 @@ public class TemplateParserContext implements ParserContext {
 
     @Override
     public String getExpressionPrefix() {
-        return "{";
+        return "{#";
```

Comment:
Is it a reg if you try with an expression like this: `{1 == #request.id}` ?

Reply:
it is :(

Reply:
After further investigation, this is not a regression compared to EL 1.9.4 behavior (before the 1.9.5 was released in 3.15.14 for this ticket : https://github.com/gravitee-io/issues/issues/8217)

I added some tests related to this syntax.
For all those cases, it's the same behavior than EL 1.9.4 (in gravitee < 3.15.14)
```
            Arguments.of("{1 == 1}", "{1 == 1}"),
            Arguments.of("{(1 == 1)}", "true"),
            Arguments.of("{(12 == 1)}", "false")
```
```
            Arguments.of("{ '/my/path/A58' == #request.pathInfo}", "{ '/my/path/A58' == #request.pathInfo}"),
            Arguments.of("{('/my/path/A58'==#request.pathInfo)}", "true"),
            Arguments.of("{('/my/path/A59'==#request.pathInfo)}", "false")
```